examples/jsm/controls/OrbitControls.js

```diff
@@ -203,6 +206,22 @@ var OrbitControls = function ( object, domElement ) {
 
 			}
 
+			// suport zoomToCursor (mouse only)
+
+			if ( scope.zoomToCursor ) {
+
+				if ( scope.object.isPerspectiveCamera ) {
+
+					scope.target.lerp( mouse3D, 1 - spherical.radius / prevRadius );
```

Comment:
`prevRadius` can be zero

Reply:
I don't think so.

Reply:
I mean, it's unlikely. 
But if you have high enough `zoomSpeed` it will.
![image](https://user-images.githubusercontent.com/3927951/62324521-1ef99f00-b480-11e9-90ea-aee1dc07afdd.png)

Then you are forever stuck in `NaN` town.

Reply:
If we set default `minDistance = EPS` then it won't ever happen.

Reply:
As we discussed offline, your concern is not a fault of this PR.